.github/workflows/ci.yml

```diff
@@ -0,0 +1,14 @@
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+      - uses: actions/cache@v1
+        with:
+          path: vendor/bundle
+          key: bundle-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: bundle
+      - run: bundle install --jobs 4 --retry 3 --deployment
+      - run: bundle exec rspec
```

Comment:
Should we also be doing [this](https://github.com/alphagov/tech-docs-monitor/pull/19/commits/05b00db7a860ea362342cb225c478c8409f6751a#diff-f9a62833538ab14fff849c2be16ab986L13)?

Reply:
It is possible to run the app locally and see what it would send, so that should be fine for local development.

Reply:
Sure, that makes sense. It's really an integration test in disguise, so if we need it, we could make it work that way instead.